runtime/oti/j9nonbuilder.h

```diff
@@ -5928,6 +5928,9 @@ typedef struct J9JavaVM {
 	jmethodID addOpens;
 	jmethodID addUses;
 	jmethodID addProvides;
+#if JAVA_SPEC_VERSION >= 19
+       jmethodID vThreadInterrupt;
```

Comment:
Could you correct the indentation here, please (it should be a tab instead of eight spaces).

Reply:
Opened: https://github.com/eclipse-openj9/openj9/pull/18969. Got it wrong while editing in vim.